tox.ini

```diff
@@ -14,29 +14,27 @@ addopts = --cov splash --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
-deps =
-	django22: Django>=2.2,<2.3
-	django30: Django>=3.0,<3.1
-	django31: Django>=3.1,<3.2
-	django32: Django>=3.2,<3.3
-	-r{toxinidir}/requirements/test.txt
-commands =
-	python -Wd -m pytest {posargs}
+deps = 
+    django32: Django>=3.2,<4.0
+    django40: Django>=3.3,<4.1
```

Comment:
this dependency should be constrained like `Django>=4.0,<4.1`

Reply:
This should be updated as per the suggestion.